app/models/form526_submission.rb

```diff
@@ -78,6 +78,13 @@ class Form526Submission < ApplicationRecord
   # plus 1 week to accomodate for edge cases and our sidekiq jobs
   MAX_PENDING_TIME = 3.weeks
 
+  # used to track in APMs between systems such as Lighthouse
+  # example: can be used as a search parameter in Datadog
+  # TODO: follow-up in ticket #93563 to make this more robust, i.e. attempts of jobs, etc.
+  def system_transaction_id
+    "Form526Submission_#{id}"
```

Comment:
Was it decided to only have the submission id in and not introduce something to differentiate between multiple tansactions (requests). IE if we retry 10 times, all would have the same ID. Would it be better to put either an attempt counter or even just random string in to differentiate the potential multiple requests? 
IE `"Form526Submission_#{id}_#{SecureRandom.uuid}"`

Reply:
this is for now. we have another ticket (mentioned in the comment there, #93563) to add the details. 
since we're in release mode right now, this is better than what it is now. 
i'd like to get this in quickly, and then iterate one it!